packages/utils/src/utils/bytecode.ts

```diff
@@ -0,0 +1,24 @@
+import type { BytesLike } from '@fuel-ts/interfaces';
+import { gzipSync, gunzipSync } from 'fflate';
+
+import { arrayify } from './arrayify';
+
+export const compressBytecode = (bytecode?: BytesLike) => {
+  if (!bytecode) {
+    return '';
+  }
```

Comment:
Could we make `bytecode` required and get rid of the conditional?

```suggestion
export const compressBytecode = (bytecode: BytesLike) => {
```

Reply:
I had to make this change because `bytecode` is an optional property on the `Abi` type:

<img width="572" alt="image" src="https://github.com/user-attachments/assets/a83d83af-a678-460e-8362-1d04b044cdb5">